src/logrotate.conf

```diff
@@ -8,6 +8,10 @@
             service ceph reload >/dev/null
         elif which invoke-rc.d > /dev/null 2>&1 && [ -x `which invoke-rc.d` ]; then
             invoke-rc.d ceph reload >/dev/null
+        elif which systemctl > /dev/null 2>&1 && [ -x `which systemctl` ]; then
+            if systemctl is-active 'ceph-*' | grep -i active > /dev/null 2>&1 ; then 
+                systemctl reload 'ceph-*' >/dev/null
```

Comment:
Do you mean `systemctl reload 'ceph-*' || :` instead of `systemctl reload 'ceph-*' >/dev/null` ?